src/FNA3D_Driver_OpenGL.c

```diff
@@ -5548,6 +5582,9 @@ static inline void CheckExtensions(
 		SDL_strstr(ext, "GL_EXT_texture_filter_anisotropic") ||
 		SDL_strstr(ext, "GL_ARB_texture_filter_anisotropic")
 	);
+	uint8_t srgbFrameBuffer = (
+		SDL_strstr(ext, "GL_EXT_framebuffer_sRGB")
```

Comment:
Did you happen to run into a hardware config where this didn't work? I'm surprised this got tripped, but at the same time we do support really old framebuffer extensions so this is probably a good idea...

Reply:
No i was merely following the pattern of the other feature detections